### PR TITLE
fix: miscellaneous updates

### PR DESCRIPTION
**Pull request by jhlee-young**

### CODE REVIEW

**Review by caramis — APPROVED** [review]
LGTM

**Review by psy2848048 — APPROVED** [review]

File: content/docs/develop/tools/xpla-js/msg-authorization.md [hunk 1]
```diff
@@ -59,10 +59,17 @@ function sendAuthorized(
 }
 
 async function main() {
+  const gasPrices = await(
+    await fetch("https://tesseract-api.xpla.dev/gas-prices", {
```
Comment (nowooj) [inline content/docs/develop/tools/xpla-js/msg-authorization.md:63]: Is it exist?

Comment (jhlee-young) [reply content/docs/develop/tools/xpla-js/msg-authorization.md:63]: nope, for now suppose it exists...🙄

Comment (nowooj) [reply content/docs/develop/tools/xpla-js/msg-authorization.md:63]: Is `tesseract-api` the url of fcd?

Comment (jhlee-young) [reply content/docs/develop/tools/xpla-js/msg-authorization.md:63]: It isn't decided yet, will update when the url is fixed

**Pull request merged**